parser/prism/Parser.h

```diff
@@ -28,7 +28,8 @@ class ParseError {
 };
 
 class Parser final {
-    // The version of Ruby syntax that we're parsing with Prism. This determines what syntax is supported or not.
+    // The version of Ruby syntax that we're parsing with Prism. This determines
+    // what syntax is supported or not.
```

Comment:
This doesn't seem to have changed the content of the comment, should it be reverted? (If your editor is setup to wrap at 80 chars, we tend to use 120 on Sorbet.)

Reply:
Ah yes, a teammate had a misconfigured editor. We rolled the indent changes back, but missed this one. Fixed!